### PR TITLE
Fixed issue #12 modal not covering whole screen in android

### PR DESCRIPTION
**Pull request by ManigandanRaamanathan**
Fixed issue #12 that statusBar space not covering whole screen in android issue.

### CODE REVIEW

**Comment by ventsi-staykov-mm** [comment]
Hi, can you please approve this PR, we really need this fix.. thanks!

File: src/components/Modal/Modal.android.tsx [hunk 1]
```diff
@@ -8,42 +8,29 @@
...
-      }
-    );
-
-    return () => {
-      backHandler.remove();
-    };
-  }, []);
-
-  return <>{visible && <View style={styles.root}>{children}</View>}</>;
+const Modal = ({ visible, children, onRequestClose, hideStatusBar }) => {
```
Comment (antonKalinin) [inline src/components/Modal/Modal.android.tsx]: Why did you decide to remove Props types?

File: src/ImageViewing.tsx [hunk 2]
```diff
@@ -47,6 +48,7 @@ function ImageViewing({
   images,
   imageIndex,
   visible,
+  hideStatusBar = true,
```
Comment (antonKalinin) [inline src/ImageViewing.tsx]: I would propose to call it
```suggestion
statusBarHidden,
```

And make it `true`  by default inside `Modal.android.tsx `

File: src/components/Modal/Modal.android.tsx [hunk 3]
```diff
@@ -8,42 +8,29 @@
...
-    backgroundColor: "transparent",
-    top: StatusBar.currentHeight
-  }
+    root: {
+        ...StyleSheet.absoluteFillObject,
+        zIndex: 1000,
+        backgroundColor: "transparent",
+        top: StatusBar.currentHeight
+    },
+    hideStatusBar: {
```
Comment (antonKalinin) [inline src/components/Modal/Modal.android.tsx]: `statusBarHidden`

**Comment by antonKalinin** [comment]
Closing this one in favour to #25

**Pull request closed without merging**